.github/workflows/main.yml

```
@@ -2,6 +2,12 @@ name: ci
 
 on: [push, pull_request]
 
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: alpine-test-${{ github.event.pull_request.number || github.ref }}
```

While this will not benefit us right now, I feel it would be better for us to use a separate concurrency group for each workflow.
```suggestion
  group: alpine-test-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

---

Thanks Prajwal! I've updated the pull request.